tests/unit/test_search_results.py

```diff
@@ -0,0 +1,75 @@
+"""Module to test the KytosGraph in graph.py."""
+from unittest import TestCase
+
+import networkx as nx
+# Core modules to import
+from kytos.core.interface import Interface
+from kytos.core.link import Link
+from kytos.core.switch import Switch
+
+# module under test
+from graph import KytosGraph
+
+
+class TestSearchResults(TestCase):
+    """Tests for the graph class."""
+
+    def setup(self):
```

Comment:
This method's name should be `setUp`, not `setup`. This way, it is automatically called before each test.

Reply:
Fixed, thank you!